firmware/application/tone_key.hpp

```diff
@@ -40,7 +43,8 @@ void tone_keys_populate(OptionsField& field);
 float tone_key_frequency(const tone_index index);
 
 std::string tone_key_string(const tone_index index);
-std::string tone_key_string_by_value(uint32_t value);
+std::string tone_key_value_string(const tone_index index);
```

Comment:
nit: not yours change, but can you remove the unnecessary `const` on value types.
I always remove them if I touch a function because they don't do anything.

Reply:
OK, removing from this module